spec/system/provider_interface/account_page_spec.rb

```diff
@@ -5,9 +5,15 @@
 
   scenario 'Provider user visits their account page with various permissions' do
     given_i_am_a_provider_user_with_dfe_sign_in
-    and_i_can_manage_organisations_but_not_users
+    given_i_cannot_manage_users_or_organisations
```

Comment:
Does this matter here?

Reply:
Because there is context around the other checks it made sense to have one for this as well.

I have now updated the notifications check to not only check for existence of notifications but also for non existence of user or organisation management links.